apps/public-docsite-v9/src/Concepts/QuickStart.stories.mdx

```diff
@@ -16,6 +16,26 @@ Fluent UI components are styled using CSS in JS. This technique requires a style
 
 Place a `<FluentProvider />` at the root of your app and pass theme as a prop.
 
+### React 18
+
+```jsx
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import { FluentProvider, teamsLightTheme } from '@fluentui/react-components';
```

Comment:
Do we want to use the `teamsLightTheme` here, or should we go with `webLightTheme` as default example?

Reply:
We should change to `webLightTheme`. Looks like the Teams theme was there before so just got copy-pasted :)